src/main/java/core/basesyntax/data/servise/FruitShopService.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.data.servise;
+
+import core.basesyntax.data.exeption.UnknownOperationException;
+import core.basesyntax.data.model.FruitTransaction;
+import core.basesyntax.data.model.OperationType;
+import core.basesyntax.data.strategy.OperationHandler;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopService {
+    private Map<String, Integer> fruitInventory = new HashMap<>();
+    private Map<OperationType, OperationHandler> operationStrategy;
```

Comment:
Consider making the `fruitInventory` and `operationStrategy` maps private and final to adhere to encapsulation principles and prevent modification outside the class.